examples/typescript/aws-s3-static-website/website/error.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<head>
+<title>Simple 404 Error Page Design</title>
+<link href="https://fonts.googleapis.com/css?family=Roboto:700" rel="stylesheet">
+<style>
+h1{
```

Comment:
Let's simplify the example page a bit and remove the styling, custom fonts, container div. Keeping it as short as the index page would be prefered.

Reply:
Fixed!